src/Raven.Server/Rachis/Leader.cs

```diff
@@ -609,7 +609,6 @@ private void CheckPromotables()
 
                 TryModifyTopology(ambassador.Key, ambassador.Value.Url, TopologyModification.Voter, out Task _);
 
-                _promotableUpdated.Set();
```

Comment:
why did you remove this?

Reply:
There was a tight loop when the task from `TryModifyTopology` wasn't completed.